src/qinterface/protected.cpp

```diff
@@ -81,23 +81,20 @@ void _expLog2x2(complex* matrix2x2, complex* outMatrix2x2, bool isExp)
         complex trace = matrix2x2[0] + matrix2x2[3];
         complex determinant = (matrix2x2[0] * matrix2x2[3]) - (matrix2x2[1] * matrix2x2[2]);
         complex quadraticRoot =
-            sqrt((matrix2x2[0] - matrix2x2[3]) * (matrix2x2[0] - matrix2x2[3]) - (real1)(4.0) * determinant);
+            sqrt(trace * trace - (real1)(4.0) * determinant);
         complex eigenvalue1 = (trace + quadraticRoot) / (real1)2.0;
         complex eigenvalue2 = (trace - quadraticRoot) / (real1)2.0;
 
-        if (norm(matrix2x2[1]) > min_norm) {
```

Comment:
Are these changes covered by the changes to unittest

Reply:
They were already covered by the unit tests run with `--disable-normalization`. The problem was, I failed to run them with normalization disabled, for a significant period of time.